misc/ls-extensions/modules/bal-shell-service/src/main/java/io/ballerina/shell/service/BalShellServiceClientCapabilities.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.ballerina.shell.service;
+
+import org.ballerinalang.langserver.commons.registration.BallerinaClientCapability;
+
+/**
+ * Client capabilities for the balShell service.
+ *
+ * @since 2201.1.1
+ */
+public class BalShellServiceClientCapabilities extends BallerinaClientCapability {
+
+    private boolean getResult;
+    private boolean getShellFileSource;
+    private boolean getVariableValues;
+
+    private boolean deleteDefined;
+    private boolean restartNotebook;
```

Comment:
Shall we rename these boolean variables using more meaning names ?

Reply:
These are named with the associated method name used in `BalShellService`.